### ISSUE TITLE
MP cis-Golgi network missing Golgi apparatus parent

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#14798]
missing 
https://www.ebi.ac.uk/QuickGO/term/GO:0005801

missing
https://www.ebi.ac.uk/QuickGO/term/GO:0005794

2012-04-19 | Deleted | RELATION | is a GO:0044431 (Golgi apparatus part)

It is part of the Golgi isn't it?

**Comment by ValWood** [comment]
yep... I can't believe I did not spot that one in 5 years!

**Comment by mah11** [comment]
I think this got discussed a bit way back in the day ... certainly it has this comment:

"The CGN is not considered part of the Golgi apparatus but is a separate organelle."

**Comment by ValWood** [comment]
Hmm, the stuff I found yesterday did not support that. so there is no general Golgi grouping term...

**Comment by hdrabkin** [comment]
Coming in at the middle of this: just make GO:0005801 as is_a to GO:0044431 and GO:0005794?

**Comment by ValWood** [comment]
Welcome to our little Golgi party  Harold!
If that is correct yes....but is it...? I hope it is for simplicity, but I don't know... I always thought that the Golgi had 2 parts, the cis-Golgi and the trans Golgi...

**Comment by ValWood** [comment]
https://en.wikipedia.org/wiki/Golgi_apparatus

In most eukaryotes, the Golgi apparatus is made up of a series of compartments consisting of two main networks: the cis Golgi network (CGN) and the trans Golgi network (TGN).

**Comment by mah11** [comment]
Wait, what? That leaves out the cisternae (aka "Golgi stack"), which is the one thing that's indisputably Golgi. Bonkers.

Back when I last did any work on the Golgi CC terms, there was disagreement in the field even about whether the TGN was really part of the Golgi apparatus or a different (but adjacent)  organelle, hence the comment still on GO:0005802. At the time, it was a bit clearer that the cis-Golgi network was considered separate, but that was at least 10, and probably more like 15, years ago. Prevailing opinion may have changed in the meantime. But I'd like to see primary literature rather than a Wikipedia article as a foundation for changing GO.

If the CGN is now considered part of the Golgi, the GO:0005801-GO:0005794 link should be part_of; if not, no link. It should not be is_a in any case.

**Comment by hdrabkin** [comment]
Right, I meant part_of of course.

**Comment by ValWood** [comment]
Everything I see looks like this
https://animalcellbiology.wordpress.com/category/cistrans-golgi/
Do we have  a reference to say the CGN is NOT part of the Golgi?

**Comment by ValWood** [comment]
So Midori do you have any recollection where the info that the CGN is not part of the Golgi came from. I could not find anything...

**Comment by mah11** [comment]
> So Midori do you have any recollection ...

No; as I said, it's been at least 10 years since I did any work on those terms.

**Comment by hdrabkin** [comment]
@ValWood  what should be done here; just add back parentage? What to whom?

**Comment by ValWood** [comment]
we need to establish if the cis-Golgi network is part of the Golgi or not.
Any experts on this around anywhere?
@pgaudet  anyone at SIB?

**Comment by hdrabkin** [comment]
I see that the network used to be an GO:0044431 (Golgi apparatus part), but this seems backwards at best
I guess I'm having problem with the term "network"  as I think it is a larger groupiing (the network would have parts?).

**Comment by pgaudet** [comment]
Well wikipedia says it is: 
https://en.wikipedia.org/wiki/Golgi_apparatus#Structure

"In most eukaryotes, the Golgi apparatus is made up of a series of compartments consisting of two main networks: the cis Golgi network (CGN) and the trans Golgi network (TGN)."

I can ask someone to confirm, but that seems like a safe assumption. 

Pascale

**Comment by ValWood** [comment]
Everything I find says it is, the British society for cell biology thinks so:
http://bscb.org/learning-resources/softcell-e-learning/golgi-apparatus/

**Comment by hdrabkin** [comment]
ok, so they are using each 'network' as a sub part of the entire golgi

**Comment by ValWood** [comment]
1) Cis Golgi network (Goods inwards)
Also called the cis Golgi reticulum it is the entry area to the Golgi apparatus. It follows the ‘transitional elements’ which are smooth areas of the RER that are also known as the ‘endoplasmic reticulum Golgi intermediate compartments’ (ERGIC).

2) Golgi stack (Main processing area) 
This section is composed of a variable number, typically 3-6, of flattened sacs called cisternae (sing. cisterna). The cisternae of the Golgi stack are divided into three working areas: cis cisternae, medial cisternae and trans cisternae.

3) trans Golgi network (Goods outwards)
This section is directly connected to the trans cisternae and it is here that final reactions and sorting takes place. The concentrated biochemicals are packed into sealed droplets or vesicles that form by budding off from the trans Golgi surface. The vesicles are then transported away for use in the cell and beyond.

**Comment by hdrabkin** [comment]
@ukemi 
 Opinion: I notice that the tgn has the relation part of golgi apparatus, the golgi stack has both part of golgi apparatus AND is a golgi apparatus part, and this one (cgn) used to have 'is a golgi apparatus part
Should each be both (is that really needed) , or should they all be just 'part of golgi'
(the non-membrane golgi terms do not have logical defs)

**Comment by ukemi** [comment]
It is not needed. If you are a golgi apparatus part, then by definition you are part of the golgi apparatus. The 'part' term is just there for is_a completeness.

**Comment by hdrabkin** [comment]
done

**Comment by raymond91125** [comment]
PMID: 35573670 The Golgi Apparatus and its Next-Door Neighbors has a different classification:
"In addition to the core layers of cisternae, the Golgi is usually accompanied by next-door compartments on its cis and trans sides. The trans-Golgi network (TGN) can be now considered as a compartment independent from the Golgi stack. On the cis side, the intermediate compartment between the ER and the Golgi (ERGIC) has been known in mammalian cells, and its functional equivalent is now suggested for yeast and plant cells."

![Image](https://github.com/user-attachments/assets/a5f10061-c22e-4661-ae36-6b35f06693d9)

**Comment by Vitalina-Kha** [comment]
Hi! I would like to include a picture from my presentation here, which illustrates the localization of different reactions of glycosphingolipid biosynthesis.
[GSL biosynthesis.pdf](https://github.com/user-attachments/files/20774405/GSL.biosynthesis.pdf)
Also, here is the paper where the authors identify three subcompartments of the Golgi apparatus: cis-Golgi network, Golgi stack and trans-Golgi network:
https://doi.org/10.1016/j.jtbi.2019.04.025
And the definition of the Golgi stack:
https://gowiki.tamu.edu/wiki/index.php/Category:GO:0005795_!_Golgi_stack
Thank you for the help.

**Comment by raymond91125** [comment]
PMID:40000850 (2025) Shows ERGIC in plant cells.

![Image](https://github.com/user-attachments/assets/3efba573-b5d2-4c19-91ed-b1b17de12d98)

PMID:21502307 (2025) Architecture of the mammalian Golgi
VTC vesicular tubular clusters ) is equivalent to ERGIC 

![Image](https://github.com/user-attachments/assets/d106b3a1-3d80-4264-b4f7-fb53c4a46e33)

**Comment by raymond91125** [comment]
@Vitalina-Kha Is ERGIC equivalent to your CGN?

Proposed hierarchy  (only showing part of relationship)
Golgi apparatus
...ERGIC/CGN
...Golgi stack
......cis-Golgi cisterna
......medial-Golgi cisterna
......trans-Golgi cisterna
...TGN

**Comment by Vitalina-Kha** [comment]
The ERGIC is not part of the cis-Golgi network, but rather a distinct compartment that acts as a bridge between the ER and the Golgi. 

PMID: [23543640](https://pubmed.ncbi.nlm.nih.gov/23543640/) 
Morpho-Functional Architecture of the Golgi Complex of Neuroendocrine Cells
"Cis and Trans Golgi Network
The Golgi stack is flanked by two tubule-vesicular networks located at the cis and trans sides, which represent the entry and exit sides of the stack, respectively (Figure 1). At the cis-side, the cis-Golgi network (CGN) is involved in ER-Golgi transport. At the trans side, the trans Golgi network (TGN) receives and packs proteins and lipids that have traversed the stack and deliver them to their final destinations.
The CGN is formed of tubules connected to the first Golgi cisterna (Sesso et al., 1994; Rambourg and Clermont, 1997). This element is well developed in some cell types such as spermatids (Vivero-Salmerón et al., 2008) but less so in others, such as prolactin cells (Rambourg and Clermont, 1997). In early electron microscopical studies, these tubules were selectively identified by using reducing osmic for prolonged times. The functional relationship of this tubular network connected to the stack (the CGN) and ER-derived pre-Golgi elements [intermediate compartment (IC), see below] remains to be established."

![Image](https://github.com/user-attachments/assets/4df1ade7-31ce-4100-8f5c-b725d581efb6)

I would propose to show them like this: 
Golgi apparatus
...ERGIC
....CGN
...Golgi stack
......cis-Golgi cisterna
......medial-Golgi cisterna
......trans-Golgi cisterna
...TGN

**Comment by raymond91125** [comment]
The distinction between ERGIC / CGN / cis-Golgi (part of the stack) is rather confusing.

[PMID:21502307](https://pmc.ncbi.nlm.nih.gov/articles/PMC3119909/) states:
"continuities between the central VTC and the cis-Golgi are regularly observed and also referred to as the cis-Golgi network". 

[PMID:9695800](https://www.cell.com/trends/cell-biology/fulltext/S0962-8924(97)01187-2) appears to show (Figure 3) that CGN is pre- medial Golgi.

[PMID:40000850
](https://www.nature.com/articles/s41556-025-01624-x) again shows (Figure 8, https://github.com/geneontology/go-ontology/issues/14798#issuecomment-2981057084) that ERGIC is the structure between ER and medial Golgi stack.

In your model, there is no ERGIC. That's why I thought perhaps your model is regarding ERGIC and CGN as one compartment.

In terms of GO, we don't need to define the terms as non-overlapping compartments. Although it is good to clarify the best we can. Thanks much for your help. @Vitalina-Kha 

BTW, note that two of the references I cited are very recent (2025) and maybe more representative of the current view of the field.

**Comment by Vitalina-Kha** [comment]
I agree that the issue of Golgi compartmentalisation is quite confusing and complex, especially concerning the intermediate compartments between the ER and the Golgi. As highlighted in the literature, the existence or non-existence of different Golgi subcompartments, as well as the quantity of Golgi stack cisternae, can vary depending on experimental approaches, the type of cells, and factors such as biological phases. Nevertheless, all these terms for the Golgi apparatus are already represented in GO for each subcompartment, and the presence of all these subcompartments can overlap with any Golgi model if necessary. The ER has terms for membrane sidedness: cytoplasmic and lumenal sides of the ER membrane. It would be helpful and logical to extend membrane bilayers to the resolution of membrane leaflets for Golgi subcompartments, in order to avoid using multiple GO terms (for membrane and membrane layer) for cellular localisation in GO-CAMs. 
Thank you for your attention to my problem @raymond91125 
 
This PMID: 21502307 (2025) is a Architecture of the mammalian Golgi. Klumperman J. Architecture of the mammalian Golgi. Cold Spring Harb Perspect Biol. 2011 Jul 1;3(7):a005181. doi: 10.1101/cshperspect.a005181. PMID: 21502307; PMCID: PMC3119909.

**Comment by Vitalina-Kha** [comment]
![Image](https://github.com/user-attachments/assets/771cecfb-6d10-40c7-8dae-bff8833b5a90)

**Comment by raymond91125** [comment]
Given that ERGIC has not been requested for annotations, we can leave it out for now.
